apps/admin/backend/src/server.ts

```diff
@@ -70,24 +51,38 @@ export async function start({
         'workspace path could not be determined; pass a workspace or run with ADMIN_WORKSPACE'
       );
     }
-    /* istanbul ignore next */
     resolvedWorkspace = createWorkspace(workspacePath);
   }
 
-  // clear any cached data
+  // Clear any cached data
   resolvedWorkspace.clearUploads();
 
-  const usb: Usb = { getUsbDrives };
-
+  let resolvedApp = app;
   /* istanbul ignore next */
-  const resolvedApp =
-    app ??
-    buildApp({
+  if (!resolvedApp) {
+    const auth = new DippedSmartCardAuth({
+      card:
+        isFeatureFlagEnabled(BooleanEnvironmentVariableName.USE_MOCK_CARDS) ||
+        isIntegrationTest()
+          ? new MockFileCard()
+          : new JavaCard(
+              constructJavaCardConfig({ includeCardProgrammingConfig: true })
+            ),
+      config: {
+        allowElectionManagersToAccessUnconfiguredMachines: false,
```

Comment:
Will this always be `false`? I don't remember running into this setting during the USB programming flow work but it's de facto always enforced. Or put another way, I don't think we allow `allowElectionManagersToAccessUnconfiguredMachines: true` because the frontend is unaware of the setting. Do we need to support that?

Reply:
So the frontend doesn't have to be aware of this setting directly - the backend handles it internally, here:

https://github.com/votingworks/vxsuite/blob/c63f0dedef27622c62c4b650225c890f3dbcff2a/libs/auth/src/dipped_smart_card_auth.ts#L617-L622

Because the setting is false in VxAdmin, using an election manager card on an unconfigured VxAdmin will surface this error screen:

https://github.com/votingworks/vxsuite/blob/c63f0dedef27622c62c4b650225c890f3dbcff2a/apps/admin/frontend/src/components/election_manager.tsx#L75-L82

Because the setting is true in VxCentralScan, on the other hand, using an election manager card on an unconfigured VxCentralScan is allowed (and actually required to configure the machine).

Also relevant is the fact that this setting is only exposed for `DippedSmartCardAuth` and not `InsertedSmartCardAuth`, which VxMark and VxScan use. On those two machines, it's understood that election managers can always access unconfigured machines.

Let me know if I can clarify any of this further!